Sources/Valet/MigrationResult.swift

```diff
@@ -42,4 +42,21 @@ public enum MigrationResult: Int, Equatable {
     case couldNotWriteToKeychain
     /// Migration failed because removing the migrated data from the keychain failed.
     case removalFailed
+
+    // MARK: CustomStringConvertible
+
+    public var description: String {
```

Comment:
adding this helped when debugging tests where the `MigrationResult` return value differed from what was expected